examples/asset/asset_loading.rs

```diff
@@ -36,13 +36,16 @@ fn setup(
     );
 
     // All assets end up in their Assets<T> collection once they are done loading:
-    if let Some(sphere) = meshes.get(&sphere_handle) {
-        // You might notice that this doesn't run! This is because assets load in parallel without
-        // blocking. When an asset has loaded, it will appear in relevant Assets<T>
-        // collection.
-        info!("{:?}", sphere.primitive_topology());
-    } else {
-        info!("sphere hasn't loaded yet");
+    match meshes.get(&sphere_handle) {
+        Some(sphere) => {
+            // You might notice that this doesn't run! This is because assets load in parallel without
+            // blocking. When an asset has loaded, it will appear in relevant Assets<T>
+            // collection.
+            info!("{:?}", sphere.primitive_topology());
+        }
+        _ => {
+            info!("sphere hasn't loaded yet");
+        }
```

Comment:
The change from `if let` to `match` is not needed.

The edition 2024 fix changes that to keep the same drop order, but we're ok with the change here